cmd/utils.go

```diff
@@ -36,22 +36,22 @@ func progress() chan bool {
 	return quit
 }
 
-func load(appID string) (*deis.Client, string, error) {
-	c, err := settings.Load()
+func load(appID string) (*deis.Client, string, int, error) {
```

Comment:
also why don't we just return the `settings` object instead of the client and limit separately?